client/state/action-types.js

```diff
@@ -109,7 +109,9 @@ export const SERVER_DESERIALIZE = 'SERVER_DESERIALIZE';
 export const SET_EXPORT_POST_TYPE = 'SET_EXPORT_POST_TYPE';
 export const SET_ROUTE = 'SET_ROUTE';
 export const SET_SECTION = 'SET_SECTION';
+// TODO(mcsf): CHANGE WORD ORDER OF THESE TWO:
```

Comment:
I wanted to keep the impact of this PR minimal, and it seemed to make
sense to do it in a PR where we change the naming in general.